recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.imx.bb

```diff
@@ -36,12 +36,15 @@ DEPENDS += "gstreamer1.0-plugins-base"
 
 inherit gobject-introspection
 
+PACKAGECONFIG_GL ?= "${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'gl', '', d)}"
+PACKAGECONFIG_GL_imxpxp = "gles2"
```

Comment:
This doesn't seem right to me. 
I don't think the PXP provides any opengl (es) support.

Reply:
We decided above to change this PR to be specific to 6UL and 6ULL, but I guess that was missed waiting for other feedback.